src/wallet/walletdb.h

```diff
@@ -252,13 +252,13 @@ class WalletBatch
     bool WriteDescriptorDerivedCache(const uint256& desc_id, uint32_t key_exp_index, uint32_t der_index, const CExtPubKey& xpub);
     bool WriteDescriptorParentCache(const uint256& desc_id, uint32_t key_exp_index, const CExtPubKey& xpub);
 
+    bool WriteActiveScriptPubKeyMan(uint8_t type, const uint256& id, bool internal);
```

Comment:
f5204ed3fe1b2a059e35e54248d38eac88c629e0 moves `WriteActiveScriptPubKeyMan` a few lines up for some reason.

Reply:
Removed it.